packages/lib/src/vis/tiles/hooks.ts

```diff
@@ -0,0 +1,13 @@
+import { useThree } from '@react-three/fiber';
+
+import { useVisibleDomains } from '../hooks';
+
+export function useDataPerPixel() {
+  const { xVisibleDomain, yVisibleDomain } = useVisibleDomains();
```

Comment:
We will discuss in the next PR but adding support for non-index coordinates would only need a call to `useValueToIndexScale` here to transform visible _data_ domains into visible _index_ domains.

But as I said, for later !